cmd/broker/ingress/main.go

```diff
@@ -32,23 +34,35 @@ import (
 	eventingv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
 	"github.com/knative/eventing/pkg/broker"
 	"github.com/knative/eventing/pkg/provisioners"
+	"github.com/knative/eventing/pkg/utils"
 	"github.com/knative/pkg/signals"
+	"go.opencensus.io/exporter/prometheus"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
 	"go.uber.org/zap"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
```

Comment:
Why `logf`?

Reply:
This is what controller-runtime calls this import. It's probably clearer to call it `crlog` or something similar. Want me to change it?

It's imported to call `logf.SetLogger(logf.ZapLogger(false))` which hooks up the internal CR logging to our zap logger.

Reply:
My preference is `crlog`. We already use `logf` in some of our e2e testing code (I think pulled from pkg). However, it is a weak preference, so do whatever you prefer.

Reply:
Done.